File: Web_search_job/ClientApp/src/app/models/backend/dtos/profiles/resume-history-work.dto.ts
import {EmployerHistoryWorkShortDTO} from "../employers/employer-history-work-short.dto";

export interface ResumeHistoryWorkDTO {
  id: number | null;
  resumeId: number;
  workName: string;
  employer: EmployerHistoryWorkShortDTO | null;
  startWorkDate: Date;
  endWorkDate: Date;
  workText: string;
}

File: Web_search_job/DTO/Resume/ResumeHistoryWorkDTO.cs
using Web_search_job.DTO.Employer;

namespace Web_search_job.DTO.Resume
{
    public class ResumeHistoryWorkDTO
    {
        public int? Id { get; set; }
        public int? ResumeId { get; set; }

        public string? WorkName { get; set; }

        public EmployerWorkExperinceShortDTO? Employer { get; set; }

        public DateTime? StartWorkDate { get; set; }
        public DateTime? EndWorkDate { get; set; }

        public string? WorkText { get; set; }
    }
}
